### PR TITLE
Make readme complete documentation

### PR DESCRIPTION
**Pull request by matishadow**

### CODE REVIEW

**Review by netcompany-martinropcke — APPROVED** [review]

**Comment by matishadow** [comment]
@MartinHolmPedersen,

If you wish to see rendered document please take a look here:
https://github.com/folkehelseinstituttet/Fhi.Smittestopp.Backend/blob/make-readme-complete-documentation/README.adoc

**Comment by OsirisTerje** [comment]
Good documentation, but why was it changed from markdown to asciidoctor ?

File: README.adoc [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+
+
+== Installation
+
+Installation process goes in the same way as with any other WebAPI project.
+For example, you could use `Web Deploy` feature of IIS but other methods work fine as well.
+
+image::readme-images/web-deploy.png[]
+
+Keep in mind that routes to all endpoint already have `api` prefix in them.
```
Comment (OsirisTerje) [inline README.adoc:23]: endpoints

File: README.adoc [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+_Remove the suggested `api` from `Application Path` when importing to not have this prefix duplicated_.
```
Comment (OsirisTerje) [inline README.adoc:24]: ..... when importing in order to .....

File: README.adoc [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+
+=== Requirements
+To run the project you need `ASP.NET Core Runtime`.
```
Comment (OsirisTerje) [inline README.adoc:27]: the ASP......

File: README.adoc [hunk 4]
```diff
@@ -0,0 +1,258 @@
...
+Any version starting with `3.1` should work fine i.e., `3.1.10`.
+Choose `Hosting Bundle` when choosing runtime version.
```
Comment (OsirisTerje) [inline README.adoc:29]: a runtime version

File: README.adoc [hunk 5]
```diff
@@ -0,0 +1,258 @@
...
+
+The project unfortunately has a couple of Windows dependencies and because of it, **you need to run it on Windows**.
```
Comment (OsirisTerje) [inline README.adoc:31]: because of that

File: README.adoc [hunk 6]
```diff
@@ -0,0 +1,258 @@
...
+To ensure that keys older than 14 days won't clutter the database,
+run the SQL command from this file link:sql/create-cleaning-sql-job.sql[].
+
+== Usage
+
+To use this `WebAPI` project you need an HTTP Client.
+Both mobile devices and frontend frameworks can use some HTTP Client to consume the API.
+
+=== Swagger
+To describe API endpoints, project makes use of Swagger.
```
Comment (OsirisTerje) [inline README.adoc:72]: the project

**Review by OsirisTerje — COMMENTED** [review]
Just noted some small spelling issues.  There are more similar ones, mostly missing articles.

**Review by OsirisTerje — APPROVED** [review]

**Pull request merged**